README.md

```diff
@@ -480,11 +480,15 @@ the following in your cloud TPU VM:
 pip install --upgrade pip
 pip install "jax[tpu]>=0.2.16" -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
 ```
+
+### pip installation: Colab TPU
 Colab TPU runtimes come with JAX pre-installed, but before importing JAX you must run the following code to initialize the TPU:
 ```python
 import jax.tools.colab_tpu
 jax.tools.colab_tpu.setup_tpu()
 ```
+Colab TPU runtimes use an older architecture than Google Cloud TPU, so installing `jax[tpu]` should be avoided on Colab.
+If for any reason you would like to update the JAX version on a Colab TPU runtime, follow the GPU-specific instructions above to install `jax[cuda11_cudnn805]`.
```

Comment:
Why install the GPU version on a TPU Colab? Shouldn't they just upgrade jax?

Reply:
I don't know whether it's necessary, but I recommended this because `cuda11_cudnn805` is the jaxlib version that comes pre-installed on Colab TPU runtimes.

Reply:
I tried it, and the CPU wheels work on TPU. I think GPU wheels are used by default because it's a one-size-fits-all solution for all accelerators (it works on CPU, GPU, and TPU).